src/main/java/org/toradocu/extractor/DocumentedMethod.java

```diff
@@ -49,7 +49,7 @@
    * return tags specified in the method's Javadoc introduced in order. (If more than one, but
    * that's weird)
    */
-  private final Set<ReturnTag> returnTags;
+  private final ReturnTag returnTags;
```

Comment:
Since the `@return` tag is only one, the field should be renamed to `returnTag`. Getter `returnTags` has to be renamed as well, and also Javadoc comments.